src/dos/dos_programs.cpp

```diff
@@ -295,23 +296,34 @@ class MOUNT : public Program {
 					temp_line = lastconfigdir + CROSS_FILESPLIT + temp_line;
 				}
 			}
+
+#if defined(WIN32)
+			/* Removing trailing backslash if not root dir so stat
+			 * will succeed */
+			if (temp_line.size() > 3 &&
```

Comment:
Is the > 3 to make sure the file is at a minimum in `x.x/` format?
Can shorter filenames exist?

Reply:
This line used to be in old code in line 304. I think the purpose is:

`path\\\\` -> `path` but `C:\` -> `C:\`.